lib/has_array_of/associated_array.rb

```diff
@@ -5,7 +5,7 @@ def self.define_in(owner_model, options)
     ids_attribute = options[:ids_attribute]
     pkey_attribute = options[:pkey_attribute]
     try_pkey_proc = options[:try_pkey_proc] = proc { |obj| obj.try(pkey_attribute) }
-    pkey_attribute_sql_type = owner_model.columns_hash[pkey_attribute.to_s].sql_type
+    pkey_attribute_sql_type = owner_model.columns_hash[pkey_attribute.to_s].sql_type  rescue "integer"
```

Comment:
what's the class of exception? please don't `rescue` from generic exception, it's not safe.